src/flame/variations/transformFunctions.ts

```diff
@@ -0,0 +1,311 @@
+import { random } from '@/shaders/random'
```

Comment:
rename this file `simpleVariations.ts`, maybe split out `dependentVariations`?

Reply:
Done, split all